lib/webdriveragent.js

```diff
@@ -399,6 +403,14 @@ class WebDriverAgent {
   set url (_url) {
     this._url = url.parse(_url);
   }
+
+  get fullyStarted () {
```

Comment:
Makes sense. We might add other conditions or effects in the future, too. I'm OK with this.